compiler/src/iree/compiler/Codegen/VMVX/LowerLinalgMicrokernels.cpp

```diff
@@ -66,65 +66,51 @@ void leftPadToRank(Location loc, SmallVectorImpl<Value> &indices,
   }
 }
 
+bool isMemRefUnitInnerStride(MemRefType type) {
+  SmallVector<AffineExpr> strideExprs;
+  AffineExpr offsetExpr;
+  if (type.getLayout().isIdentity()) {
+    return true;
+  }
+
+  if (failed(mlir::getStridesAndOffset(type, strideExprs, offsetExpr))) {
```

Comment:
There is an API entry that has the signature `(MemRefType type, SmallVector<int64_t> &, int64_t)` that is probably more suited for what is being done here.

Reply:
Ah, I see from the impl that returns kDynamicStrideOrOffset for dynamic strides. I misread it and thought it was failing in that case.